pkg/render/logstorage/dashboards/dashboards.go

```diff
@@ -148,11 +152,33 @@ func (d *dashboards) resources() []client.Object {
 func (d *dashboards) AllowTigeraPolicy() *v3.NetworkPolicy {
 	egressRules := []v3.Rule{}
 	egressRules = networkpolicy.AppendDNSEgressRules(egressRules, d.cfg.Installation.KubernetesProvider == operatorv1.ProviderOpenShift)
-	egressRules = append(egressRules, v3.Rule{
-		Action:      v3.Allow,
-		Protocol:    &networkpolicy.TCPProtocol,
-		Destination: render.KibanaEntityRule,
-	})
+	if d.cfg.ExternalKibanaClientSecret != nil {
+		const defaultKibanaPort uint64 = 443
+		kibanaDomain := strings.TrimPrefix(d.cfg.KibanaHost, "https://")
+		kibanaDomain = strings.TrimSuffix(kibanaDomain, "/")
+		kibanaPort, err := strconv.ParseUint(d.cfg.KibanaPort, 10, 16)
+		if err != nil {
+			kibanaPort = defaultKibanaPort
```

Comment:
We definitely need to log out if there is an error here

Reply:
If we need to support "no port specified" then we should determine that by parsing the string and deciding that there was no port given - we want to be able to tell when `ParseUint` fails for a legitimate reason.